content/en/spec/activitypub.md

```diff
@@ -29,21 +25,21 @@ Like
 Announce
 : Transformed into a boost on a status
 
-Flag
-: Transformed into a report to the moderation team.
-
 Update
-: Refresh vote count on polls. As of Mastodon 3.5.0, can be used to edit statuses when the `updated` timestamp is present.
+: Refresh vote count on polls. As of Mastodon 3.5.0: edit statuses.
```

Comment:
Why did this line change?

Reply:
i'm... not sure. i think i could go back and add details to all these activities.